src/commands/CreateVDBCommand.ts

```diff
@@ -53,17 +53,17 @@ export function createVDBCommand() {
 	}
 }
 
-function handleVDBCreation(filepath: string, fileName: string): Promise<boolean> {
+export function handleVDBCreation(filepath: string, fileName: string, templateFolder?: string): Promise<boolean> {
 	return new Promise<boolean>( (resolve, reject) => {
 		if (fileName && fileName.length>0) {
 			try {
-				let templatePath = path.join(extension.pluginResourcesPath, "vdb_template.yaml");
+				let templatePath = templateFolder ? path.join(templateFolder, "vdb_template.yaml") : path.join(extension.pluginResourcesPath, "vdb_template.yaml");
```

Comment:
I think it would be best to remove the logic from here and force passing the templateFolder for all callers.

Also I don't understand why we have 2 different cases. We want to tets with different templates in test? Why?